cvat-data/README.md

```diff
@@ -0,0 +1,7 @@
+# cvat-data module
+
+```bash
+npm run build  # build with minification
```

Comment:
Need to make the command line compatible with the current version of cvat-ui (`npm build`, `npm start`).